google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -243,6 +287,252 @@ void DeleteCluster(google::cloud::bigtable::InstanceAdmin instance_admin,
+        zone_1, 3, google::cloud::bigtable::ClusterConfig::HDD);
+    google::cloud::bigtable::InstanceConfig config(
+        google::cloud::bigtable::InstanceId(instance_id), display_name,
+        {{cluster_id_1.get(), cluster_config}});
+    config.set_type(google::cloud::bigtable::InstanceConfig::PRODUCTION);
+
+    auto future = instance_admin.CreateInstance(config).get();
+    std::cout << " Done" << std::endl;
+    // [END bigtable_create_prod_instance]
+  } else {
+    std::cout << std::endl
+              << "Instance " << instance_id.get() << " already exists."
+              << std::endl;
+    return;
+  }
+
+  // [START bigtable_list_instance]
+  // List Instances
+  std::cout << std::endl << "Listing Instances: " << std::endl;
+  auto instances_after = instance_admin.ListInstances();
+  for (auto const& instance : instances_after) {
+    std::cout << instance.name() << std::endl;
+  }
+  // [END bigtable_list_instance]
+
+  // [START bigtable_get_instance]
+  // get Instance
+  std::cout << std::endl << "Get Instance: " << std::endl;
+  auto instance = instance_admin.GetInstance(instance_id.get());
+  std::string instance_detail;
+  google::protobuf::TextFormat::PrintToString(instance, &instance_detail);
+  std::cout << "GetInstance details : " << std::endl << instance_detail;
+  // [END bigtable_get_instance]
+
+  // [START bigtable_list_cluster]
+  // List clusters
+  std::cout << std::endl << "Listing Clusters: " << std::endl;
+  auto cluster_list = instance_admin.ListClusters(instance_id.get());
+  std::cout << "Cluster Name List: " << std::endl;
+  for (auto const& cluster : cluster_list) {
+    std::cout << "Cluster Name: " << cluster.name() << std::endl;
+  }
+  // [END bigtable_list_cluster]
+
+  // [START bigtable_create_cluster]
+  std::cout << std::endl << "Adding Cluster: " << std::endl;
+  auto location =
+      "projects/" + instance_admin.project_id() + "/locations/" + zone_2;
+  auto cluster_config = google::cloud::bigtable::ClusterConfig(
+      location, 3, google::cloud::bigtable::ClusterConfig::HDD);
+  auto cluster =
+      instance_admin.CreateCluster(cluster_config, instance_id, cluster_id_2);
+  std::cout << "Cluster Created " << cluster.get().name() << std::endl;
+  // [END bigtable_create_cluster]
+
+  // [START bigtable_delete_cluster]
+  // Delete Cluster
+  std::cout << std::endl << "Deleting Cluster: ";
+  instance_admin.DeleteCluster(instance_id, cluster_id_2);
+  std::cout << " Done" << std::endl;
+  // [END bigtable_delete_cluster]
+
+  // [START bigtable_delete_instance]
+  // Delete Instance
+  std::cout << std::endl << "Deleting Instance: ";
+  instance_admin.DeleteInstance(instance_id.get());
+  std::cout << " Done" << std::endl;
+  // [END bigtable_delete_instance]
+}
+
+// This full example demonstrate various instance operations
+// by initially creating development instance
+void RunFullExample2(std::string project_id, int argc, char* argv[]) {
```

Comment:
Why do we have a second function that does basically the same?  I mean, I do not mind, just want to understand.

Reply:
The second example is for development instance, Node.js has global marker tags for both development and production instance so I created couple of examples and used global marker tags there. Full example also shows that `CreateCluster` is not possible with development instance.

Reply:
We don't need a full example.  We just need the `bigtable_create_dev_instance` section.  This should be `CreateDevInstane` instead of `RunFullExample2`.